grpc/src/types/tx.rs

```diff
@@ -0,0 +1,245 @@
+use std::convert::Infallible;
+
+use cosmrs::Tx;
+use k256::ecdsa::{signature::Signer, Signature};
+use prost::{Message, Name};
+use serde::{Deserialize, Serialize};
+use pbjson_types::Any;
+
+use celestia_proto::cosmos::base::abci::v1beta1::{AbciMessageLog, TxResponse as RawTxResponse};
+use celestia_proto::cosmos::base::v1beta1::Coin;
+use celestia_proto::cosmos::crypto::secp256k1;
+use celestia_proto::cosmos::tx::v1beta1::mode_info::{Single, Sum};
+use celestia_proto::cosmos::tx::v1beta1::{
+    AuthInfo, BroadcastMode, BroadcastTxRequest, BroadcastTxResponse, Fee,
+    GetTxRequest as RawGetTxRequest, GetTxResponse as RawGetTxResponse, ModeInfo, SignDoc,
+    SignerInfo, Tx as RawTx, TxBody,
+};
+use celestia_tendermint_proto::v0_34::abci::Event;
+use celestia_tendermint_proto::Protobuf;
+use celestia_types::auth::{AccountKeypair, BaseAccount};
+use celestia_types::blob::{Blob, MsgPayForBlobs, RawBlob, RawBlobTx, RawMsgPayForBlobs};
+
+use crate::types::{FromGrpcResponse, IntoGrpcParam};
+use crate::Error;
+
+/// Response to a tx query
+#[derive(Debug, Clone, Serialize, Deserialize)]
```

Comment:
we have `celestia_types::state::TxResponse`, I think we should either move this there to types or just use the one from types (although it's just re-export)

Reply:
Moved the transaction related types to celestia types and updated already existing ones :+1: